homeassistant/components/atag/__init__.py

```diff
@@ -188,24 +91,21 @@ async def async_unload_entry(hass, entry):
 class AtagEntity(Entity):
     """Defines a base Atag entity."""
 
-    def __init__(self, coordinator: AtagDataUpdateCoordinator, atag_type: dict) -> None:
+    def __init__(self, coordinator: AtagDataUpdateCoordinator, id: str) -> None:
```

Comment:
`id` is a builtin function. I'd name this variable `domain` since that's what we're passing.

Reply:
Its not really the domain (as it also identifies the sensortype), but atag_id should work

Reply:
> Please avoid a breaking change if possible by updating existing entry data in `async_setup_entry`.

Just confirmed that the config entry itself doesn't need to be adjusted, as I've made it an optional parameter, which doesn't cause a break if its missing. 
That only leaves the entity_ids of the sensor platform that would be renamed, how should I go about that?

Thanks for helping out, this should be the last big change to the integration :)